### PR TITLE
Clean up of docker swarm service

### PR DESCRIPTION
**Pull request by mscherer**
##### SUMMARY
Cleanup of docker swarm service found by https://lgtm.com

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
docker_swarm_service

### CODE REVIEW

File: lib/ansible/modules/cloud/docker/docker_swarm_service.py [hunk 1]
```diff
@@ -1069,32 +1069,29 @@ def run(self):
...
                         msg = 'Service updated'
                         rebuilt = False
-                        changes = changes
                 else:
-                    if force_update and not module.check_mode:
-                        self.update_service(module.params['name'],
-                                            current_service,
-                                            new_service)
+                    if force_update:
+                        if module.check_mode:
```
Comment (dariko) [inline lib/ansible/modules/cloud/docker/docker_swarm_service.py]: Shouldn't it be
`````
if force_update:
    if not module.check_mode:
`````
here?

Comment (mscherer) [reply lib/ansible/modules/cloud/docker/docker_swarm_service.py]: Indeed, my bad. I knew I should have done that _after_ sleeping.

**Review by jwitko — APPROVED** [review]

**Comment by jwitko** [comment]
shipit

**Review by felixfontein — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
bot_status

**Pull request merged**